server/controllers/UserLoginController.js

```diff
@@ -0,0 +1,44 @@
+import bcrypt from 'bcryptjs';
+import Token from '../token';
+import database from '../models/database';
+
+class UserLoginController {
+  static userLogin(req, res) {
+    /**
+   * @description Method to login a user
+   *
+   * @param {Object} UserLoginController
+   *
+   * @return {JSON Object} Returns JSON obejects of responses
+   */
+    const results = [];
+    database.query(`SELECT id, role, password FROM users WHERE email= '${req.body.email}'`, (err, response) => {
+      if (response.rows.length <= 0) {
```

Comment:
Why are using **<=**... is it possible to have a response length that is less than 1?